switools/swisigningservice.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python3.7
+import os
```

Comment:
If you do what I suggest below, then remove `import os` and add:
```suggestion
import re
import subprocess
```

Reply:
we still need os.system to insert signature files into the swi. zipfile.py is very nasty in the meta information it adds to the zip entry (like version required to extract = 2.0, and that meta info, like the timestamp, affects the computation of the checksum/signature).

Reply:
I think you've added this comment in the wrong place - this file doesn't touch the SWI at all 😃 

Also, I still think it would be better to use `subprocess` here too.